tests/func_test.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+
+import ddt
+
+from src.Homework6 import homework4_2
+
+
+@ddt.ddt
+class TestHomework(unittest.TestCase):
+
+    @ddt.data(
+        (('Moscow',), ['Moscow - Russia']),
+        (('Brest',), ['Brest - Belarus', 'Brest - France']),
+        (('Kiev',), ['Kiev - Ukraine']),
+        (('Kiev', 'Minsk'), ['Kiev - Ukraine', 'Minsk - Belarus']),
+        ((12312, '12312'), []),
+        (['Moscow', 'Kiev'], ['Moscow - Russia', 'Kiev - Ukraine']),
+        ({'Moscow': '123', 'Kiev': 'sada'}, ['Moscow - Russia',
+                                             'Kiev - Ukraine']),
+    )
+    @ddt.unpack
+    def test_cities(self, args, expected):
```

Comment:
missed test setup. it shouldn't be the same. we should also test different setup.